### DISCUSSION TITLE
Docker Desktop for windows  Update budibase to latest

### DISCUSSION THREAD

**Discussion by bigfoot53** [Budibase/budibase#9989] [Help]
Evening everyone, 
Im running Docker Desktop for windows and would like to update to the latest build . How do i do this ?
I tried  pull and got a second image and now the old one says dangling,  for a breif time i couldnt get back int my db but after a restart it allows me back into the old db  not updated , 

Thanks
Bruce

**Comment by melohagan** [comment]
Hey @bigfoot53 

An easy way to manage upgrading Budibase is to use the CLI: https://docs.budibase.com/docs/budibase-cli-setup

Then you can simply run the `budi hosting --update` command.

With that said, if you are using **docker-compose** you should be able to simply do a `docker-compose down` followed by a `docker-compose up -d`

**Comment by bigfoot53** [comment]
![image](https://user-images.githubusercontent.com/61506151/224872952-3ef99c01-b5fb-4183-8478-70338faea77f.png)

I tried both options from the terminal in docker and get errors shown in pic

**Reply by melohagan** [reply]
I would run either of those commands in the terminal outside of your docker container. 

You will also need to `cd` into the folder that contains your **docker-compose.yaml** file.

---

**1. Folder containing docker-compose file**
![Screenshot 2023-03-14 at 08 09 27](https://user-images.githubusercontent.com/101575380/224936265-50b2f198-b3b8-4881-a202-8d537a2a7152.png)

**2. Navigating from user root to the folder that contains my docker-compose file**
![Screenshot 2023-03-14 at 08 10 11](https://user-images.githubusercontent.com/101575380/224936428-3a3e7b82-bc7c-4cf7-b91d-cc7cd665abc5.png)

**3. Running the command**
![Screenshot 2023-03-14 at 08 10 49](https://user-images.githubusercontent.com/101575380/224936586-7365e839-86f3-4464-82aa-83b9f6485160.png)
